src/chesster.js

```diff
@@ -970,3 +972,87 @@ function(bot, message) {
+},
+function(bot, message) {
+    var deferred = Q.defer();
+    bot.startPrivateConversation(message, function (response, convo) {
+        subscription.processTellCommand(chesster.config, message).then(function(response) {
+            convo.say(response);
+            deferred.resolve();
+        }).catch(function(error) {
+            convo.say("I'm sorry, but an error occurred processing this subscription command");
+            deferred.reject(error);
+        });
+    });
+    return deferred.promise;
+});
+
+chesster.hears({
+    middleware: [],
+    patterns: ['^subscriptions$'],
+    messageTypes: ['direct_message']
+},
+function(bot, message) {
+    var deferred = Q.defer();
+    bot.startPrivateConversation(message, function (response, convo) {
+        subscription.processSubscriptionsCommand(chesster.config, message).then(function(response) {
+            convo.say(response);
+            deferred.resolve();
+        }).catch(function(error) {
+            convo.say("I'm sorry, but an error occurred processing this subscription command");
+            deferred.reject(error);
+        });
+    });
+    return deferred.promise;
+});
+
+chesster.hears({
+    middleware: [],
+    patterns: [/^remove subscription (\d+)$/],
+    messageTypes: ['direct_message']
+},
+function(bot, message) {
+    var deferred = Q.defer();
+    bot.startPrivateConversation(message, function (response, convo) {
+        subscription.processRemoveSubscriptionCommand(chesster.config, message, message.match[1]).then(function(response) {
+            convo.say(response);
+            deferred.resolve();
+        }).catch(function(error) {
+            convo.say("I'm sorry, but an error occurred processing this subscription command");
+            deferred.reject(error);
+        });
+    });
+    return deferred.promise;
+});
+
+subscription.emitter.on('a-game-is-scheduled', function(bot, message, league, results, white, black) {
+    // TODO: encase this in a exception Handler
+    _.each([white.name, black.name], function(source) {
+        // TODO: this will also need to deal with channels at some point
+        subscription.getListeners(league.options.name, source, 'a-game-is-scheduled').then(function(targets) {
+            _.each(targets, function(target) {
+                target = player.getSlackUserFromNameOrID(slack.users, target);
+                if (!_.isUndefined(target)) {
+                    bot.startPrivateConversation({user: target.id}, function(err, convo) {
+                        // TODO: this could be a much better message!
+                        convo.say("{white} vs {black} has been scheduled for {date}.".format({
+                            white: white.name,
+                            black: black.name,
+                            date: "TODO"
+                        }));
+                    });
+                } else {
+                    // TODO: probably delete the subscription at this point.
```

Comment:
Yeah how would this get triggered exactly?  The user no longer exists in slack?  If that's the case, then the subscription should be deleted as this comment indicates.